src/traits/election/vrf.rs

```diff
@@ -282,17 +284,17 @@ where
 
 /// the vrf implementation
 #[derive(Derivative)]
-#[derivative(Eq, PartialEq)]
+#[derivative(Debug, Eq, PartialEq)]
 pub struct VrfImpl<TYPES, LEAF: LeafType<NodeType = TYPES>, SIGSCHEME, VRF, VRFHASHER, VRFPARAMS>
 where
     VRF: Vrf<VRFHASHER, VRFPARAMS> + Sync + Send,
     TYPES: NodeType,
 {
     /// the stake table
-    #[derivative(PartialEq = "ignore")]
+    #[derivative(Debug = "ignore", PartialEq = "ignore")]
```

Comment:
Printing the stake table is most likely useful for debugging. But, we no longer care about vrf based election (afaik?) so I don't think this matters.